packages/paste-website/src/components/AspectRatio.tsx

```diff
@@ -0,0 +1,57 @@
+/** @jsx jsx */
```

Comment:
Pretty sure I fixed this to work without the jsx pragma stuff

Reply:
Yep. Looks like you did. Removing it.